GETTING_STARTED.md

```diff
@@ -1,12 +1,13 @@
 ## Getting started
 
 1. Install [Node 16 LTS](https://nodejs.org/en/)
-2. Install and get familiar with [Git](https://docs.github.com/en/get-started/quickstart/hello-world)
+2. Install [Git](https://git-scm.com/downloads)
+3. Install and get familiar with [Git](https://docs.github.com/en/get-started/quickstart/hello-world)
```

Comment:
I would also personally remove the "Install and" since it is already mentioned in the previous step to install Git

Reply:
Thank you so much!